dll/win32/devmgr/properties/advprop.cpp

```diff
@@ -1909,6 +1936,51 @@ AdvProcDetailsDlgProc(IN HWND hwndDlg,
                                                     GetDlgItem(hwndDlg, IDC_DETAILSPROPVALUE));
                         }
                         break;
+
+                    case IDS_MENU_COPY:
+                    {
+                        HWND hwndListView = GetDlgItem(hwndDlg, IDC_DETAILSPROPVALUE);
+                        INT nSelectedItems = ListView_GetSelectedCount(hwndListView);
+                        INT nSelectedId = ListView_GetSelectionMark(hwndListView);
+
+                        if (nSelectedId >= 0 && nSelectedItems > 0)
+                        {
+                            TCHAR szItemName[MAX_PATH];
+                            HGLOBAL hGlobal;
+                            LPWSTR pszBuffer;
+
+                            ListView_GetItemText(hwndListView,
+                                                 nSelectedId, 0,
+                                                 szItemName,
+                                                 MAX_PATH);
+
+                            hGlobal = GlobalAlloc(GHND, MAX_PATH);
+                            pszBuffer = (LPWSTR)GlobalLock(hGlobal);
+
+                            if (pszBuffer == NULL)
+                            {
+                                GlobalFree(hGlobal);
+                                return Ret;
+                            }
+
+                            wsprintfW(pszBuffer, L"%s", szItemName);
```

Comment:
```suggestion
                            wsprintf(pszBuffer, L"%s", szItemName);
```
or use a StringSafe function instead.